elytron/src/main/java/org/wildfly/extension/elytron/AuditResourceDefinitions.java

```diff
@@ -240,14 +240,16 @@ static ResourceDefinition getRotatingFileAuditLogResourceDefinition() {
                     final SecurityEventVisitor<?, String> formatter = Format.JSON == format ? JsonSecurityEventFormatter.builder().setDateFormatSupplier(bind(SimpleDateFormat::new, DATE_FORMAT)).build() : SimpleSecurityEventFormatter.builder().setDateFormatSupplier(bind(SimpleDateFormat::new, DATE_FORMAT)).build();
                     AuditEndpoint endpoint;
                     try {
-                        endpoint = RotatingFileAuditEndpoint.builder()
+                        FileAuditEndpoint.Builder builder = RotatingFileAuditEndpoint.builder()
```

Comment:
Sorry, I missed this one yesterday. Is there a reason the type isn't `RotationgFileAuditEndpoint.Builder`? It would save the casting needed if the `suffix` is defined.